backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/map/BpkIconMapMarker.kt

```diff
@@ -67,9 +70,9 @@ fun BpkIconMapMarker(
         tag = tag,
         title = contentDescription,
         visible = visible,
-        zIndex = zIndex,
+        zIndex = if (status == BpkIconMarkerStatus.Focused && zIndex == null) 1.0f else zIndex ?: 0.0f,
```

Comment:
this sets the zIndex to 1 for a focused marker to ensure it shows on top of the others